sample/captor/service_test.go

```diff
@@ -0,0 +1,31 @@
+package captor
```

Comment:
Can you please move this to `mockgen/internal/tests`.  This sample folder should probably never have been located where it is. As is this technically leaks into the public api.